d3a_api_client/setups/test_create_aggregator.py

```diff
@@ -53,10 +56,12 @@ def on_finish(self, finish_info):
         self.is_finished = True
 
 
-import os
-simulation_id = os.environ["API_CLIENT_SIMULATION_ID"]
-domain_name = os.environ["API_CLIENT_DOMAIN_NAME"]
-websocket_domain_name = os.environ["API_CLIENT_WEBSOCKET_DOMAIN_NAME"]
+try:
+    simulation_id = os.environ["API_CLIENT_SIMULATION_ID"]
+    domain_name = os.environ["API_CLIENT_DOMAIN_NAME"]
+    websocket_domain_name = os.environ["API_CLIENT_WEBSOCKET_DOMAIN_NAME"]
+except KeyError as ex:
+    sys.exit(f'Missing environment variable: {ex.args[0]}. The script cannot continue.')
```

Comment:
Added a helper method(https://github.com/gridsingularity/d3a-api-client/pull/94/files#diff-648e3accb4fdc60dfb72236a6e64c3c56ca48d9d6f98297eed6bf2bc6d8af90bR282) to get these values either via cli argument or via environment variable BUT will further to catch key error too